### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast :The color contrast ratio of keyboard focus on several controls such as "Azure serverless community library" is less than 3:1

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#104553]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags: #A11y_VSCode;#Visual Studio Code;#CAITestPass_May2020;#A11yTCS;#A11yWCAG2.1;#MAC;#FTP;#Benchmark;#ColorContrast;#A11ySev2:#MAS1.4.11;

**Environment Details:**
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.4.11

**Repro Steps:**
Open Visual studio code
Click on extension button and search for "Azure Functions"
Click on install and move to the "Azure" button, the "Default Screen" will appear.
Navigate to several controls such as "Azure serverless community library" using keyboard and verify if the color contrast ratio of keyboard focus is less than 3:1 or not.

**Actual:**
The several controls such as "Azure serverless community library" has keyboard focus color contrast ratio of 1.5:1 which is less than the standard color contrast ratio of 3:1 which is not appropriate as the users with low vision will have difficulty in understanding the navigation on the screen.

**Expected:**
The keyboard focus color contrast ratio of "Azure serverless community library" control should be greater than or equal to 3:1.

**Attachment link for Reference:**
<img width="1403" alt="104553_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast" src="https://user-images.githubusercontent.com/53940787/90130978-f988bc00-dd88-11ea-83de-00862872d4f1.png">

**Comment by isidorn** [comment]
Please file against ms-azuretools.vscode-azurefunctions extension.
Then that team can identify if it is really an issue in vscode and file it here with minimal human readable steps.

**Comment by nirmalsahoo** [comment]
@isidorn We have already logged bug in  ms-azuretools.vscode-azurefunctions extension and their team says that this issue is external to VSCode then we have filed issue .

Bug reference: https://github.com/microsoft/vscode-azurefunctions/issues/2283
Please reactivate this issue.

**Comment by isidorn** [comment]
Makes sesne but that team should then file an issue in VS Code. They should put effort in filling an issue and use VS Code language when filling an issue, not just forward you to file an issue in our repository. They should search for duplicates etc..
